### PR TITLE
AnyNiivue to Niivue

### PR DESCRIPTION
**Pull request by kolibril13**
AnyNiivue was just a random choise in the first place, and I think that Niivue is the better name for the widget class.

### CODE REVIEW

**Comment by christian-oreilly** [comment]
@kolibril13 I think, if you are making this change, you should use NiiVue, not Niivue, so that it aligns with how the name of the parent project is capitalized. Else, it will be a bit awkward.

File: README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ pip install ipyniivue
 In a Jupyter environment:
 
 ```py
-from ipyniivue import AnyNiivue
+from ipyniivue import Niivue
```
Comment (kolibril13) [inline README.md:17]:
```suggestion
from ipyniivue import NiiVue
```

File: README.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ pip install ipyniivue
...
 
-nv = AnyNiivue()
+nv = Niivue()
```
Comment (kolibril13) [inline README.md:19]:
```suggestion
nv = NiiVue()
```

File: demo/additive_voxels.ipynb [hunk 3]
```diff
@@ -18,7 +18,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from ipyniivue import AnyNiivue, SliceType\n",
+    "from ipyniivue import Niivue, SliceType\n",
```
Comment (kolibril13) [inline demo/additive_voxels.ipynb:21]:
```suggestion
    "from ipyniivue import NiiVue, SliceType\n",
```

File: demo/additive_voxels.ipynb [hunk 4]
```diff
@@ -36,7 +36,7 @@
     "        \"cal_max\": 4,\n",
     "    },\n",
     "]\n",
-    "nv = AnyNiivue(\n",
+    "nv = Niivue(\n",
```
Comment (kolibril13) [inline demo/additive_voxels.ipynb:39]:
```suggestion
    "nv = NiiVue(\n",
```

File: demo/basic_multiplanar.ipynb [hunk 5]
```diff
@@ -17,15 +17,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from ipyniivue import AnyNiivue, SliceType\n",
+    "from ipyniivue import Niivue, SliceType\n",
```
Comment (kolibril13) [inline demo/basic_multiplanar.ipynb:20]:
```suggestion
    "from ipyniivue import NiiVue, SliceType\n",
```

File: demo/basic_multiplanar.ipynb [hunk 6]
```diff
@@ -17,15 +17,15 @@
...
     "# based on https://niivue.github.io/niivue/features/basic.multiplanar.html\n",
     "\n",
     "volumes = [\n",
     "  { \"path\": \"../images/mni152.nii.gz\", \"colormap\": \"gray\", \"visible\": True, \"opacity\": 1.0 },\n",
     "  { \"path\": \"../images/hippo.nii.gz\", \"colormap\": \"red\", \"visible\": True, \"opacity\": 1 },\n",
     "\n",
     "]\n",
-    "nv = AnyNiivue(slice_type=SliceType.MULTIPLANAR)\n",
+    "nv = Niivue(slice_type=SliceType.MULTIPLANAR)\n",
```
Comment (kolibril13) [inline demo/basic_multiplanar.ipynb:28]:
```suggestion
    "nv = NiiVue(slice_type=SliceType.MULTIPLANAR)\n",
```

File: src/ipyniivue/__init__.py [hunk 7]
```diff
@@ -3,6 +3,6 @@
 import importlib.metadata
 
 from ._constants import DragMode, MuliplanarType, SliceType  # noqa: F401
-from ._widget import AnyNiivue  # noqa: F401
+from ._widget import Niivue  # noqa: F401
```
Comment (kolibril13) [inline src/ipyniivue/__init__.py:6]:
```suggestion
from ._widget import NiiVue  # noqa: F401
```

File: src/ipyniivue/_widget.py [hunk 8]
```diff
@@ -8,7 +8,7 @@
 from ._options_mixin import OptionsMixin
 from ._utils import file_serializer, serialize_options, snake_to_camel
 
-__all__ = ["AnyNiivue"]
+__all__ = ["Niivue"]
```
Comment (kolibril13) [inline src/ipyniivue/_widget.py:11]:
```suggestion
__all__ = ["NiiVue"]
```

File: src/ipyniivue/_widget.py [hunk 9]
```diff
@@ -22,7 +22,7 @@ class Volume(ipywidgets.Widget):
     cal_max = t.Float(None, allow_none=True).tag(sync=True)
 
 
-class AnyNiivue(OptionsMixin, anywidget.AnyWidget):
+class Niivue(OptionsMixin, anywidget.AnyWidget):
```
Comment (kolibril13) [inline src/ipyniivue/_widget.py:25]:
```suggestion
class NiiVue(OptionsMixin, anywidget.AnyWidget):
```

**Comment by kolibril13** [comment]
> you should use NiiVue, not Niivue, so that it aligns with how the name of the parent project is capitalized

thanks @christian-oreilly , good catch!
I've renamed the class accordingly!

**Review by manzt — APPROVED** [review]
Looks great

**Pull request merged**